### ISSUE TITLE
Incorrectly flagged Possible method call error

### ISSUE THREAD

**Issue by chriswaudby** [julia-vscode/StaticLint.jl#354]

**Comment by bEiGeOnE78** [comment]
I have a similar issue with Plots. It only happens when I add a new line after a plot() or plot!() call. In the code below. In the code below plot() has the blue squiggles. plot!() does not. If I add a new line after plot!() it will get highlighted.

`using Plots
plotlyjs()

x = range(0, 10, length=100)
y1 = sin.(x)
y2 = cos.(x)
p = plot(x, [y1 y2])

y3 = @. sin(x)^2 - 1/2
plot!(p, x, y3)`

**Comment by axsk** [comment]
Not sure if this is the same problem, but the package I'm working on requires only `RecipesBase`. There `plot` has 0 methods implemented, hence all of my `plot` calls (dozens) are squiggly, rendering the Problems pane rather useless.

**Comment by leespen1** [comment]
I'm having the same issue.
```
    pl = plot(title = "Title", xlabel="# Steps", ylabel="Error", scale=:log10)
```
Is all blue-squiggly underlined.


Julia Version: 1.9.0
Julia VSCode Extension: 1.47.2
VSCode Version: 1.79.2
OS: Ubuntu 22.04.2 LTS

**Comment by jonathan-conder-sm** [comment]
I've seen this as well with `Docs.doc(::Docs.Binding)`. Think it's because `Docs.doc` is defined in `Base` but all of its methods are implemented in `REPL`.
